src/components/exchange/ExchangeNotch.tsx

```diff
@@ -1,36 +1,30 @@
 import React from 'react';
 import FastImage from 'react-native-fast-image';
-import { useTheme } from '../../theme/ThemeContext';
-import { Row } from '../layout';
 import ExchangeNotchLeft from '@rainbow-me/assets/exchangeNotchLeft.png';
 import ExchangeNotchLeftDark from '@rainbow-me/assets/exchangeNotchLeftDark.png';
 import ExchangeNotchMiddle from '@rainbow-me/assets/exchangeNotchMiddle.png';
 import ExchangeNotchMiddleDark from '@rainbow-me/assets/exchangeNotchMiddleDark.png';
 import ExchangeNotchRight from '@rainbow-me/assets/exchangeNotchRight.png';
 import ExchangeNotchRightDark from '@rainbow-me/assets/exchangeNotchRightDark.png';
+import { Box } from '@rainbow-me/design-system';
 import { useDimensions } from '@rainbow-me/hooks';
 import styled from '@rainbow-me/styled-components';
+import { useTheme } from '@rainbow-me/theme';
 
 const notchHeight = 48;
 const notchSideWidth = 78;
 const ANDROID_NOTCH_OFFSET = 8;
 
-const Container = styled(Row).attrs({
-  pointerEvents: 'none',
-})({
-  height: notchHeight,
-  left: 0,
-  position: 'absolute',
-  top: 132,
-});
-
-const NotchMiddle = styled(FastImage).attrs(({ isDarkMode }) => ({
-  resizeMode: FastImage.resizeMode.stretch,
-  source: isDarkMode ? ExchangeNotchMiddleDark : ExchangeNotchMiddle,
-}))({
+const NotchMiddle = styled(FastImage).attrs(
```

Comment:
Can we migrate this to DS? `<Box as={FastImage} ...>`

Reply:
Box doesnt seem to like passing in FastImage as the `as` prop. The props dont seem to apply to it.